### PR TITLE
Add jlord

### PR DESCRIPTION
**Pull request by AromaR**

### CODE REVIEW

**Comment by jscottmorse** [comment]
Comment

File: contributors/add-lovrozekusic.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+lovrozekusic
```
Comment (ghost) [inline contributors/add-lovrozekusic.txt:1]: testing

**Comment by willshion** [comment]
ha ve a com mont

File: contributors/add-lovrozekusic.txt (hunk 1 above)
Comment (willshion) [inline contributors/add-lovrozekusic.txt:1]: fawefaw

**Pull request closed without merging**